win/libsass.vcxproj

```diff
@@ -1,17 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="GitVersion;Main"
-    xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-
+<Project ToolsVersion="14.0" DefaultTargets="GitVersion;Main" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <SASSC_VERSION>[NA]</SASSC_VERSION>
     <LIBSASS_VERSION>[NA]</LIBSASS_VERSION>
   </PropertyGroup>
-
   <Target Name="GitVersion">
-
-    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags > libsass-version" />
-    <Exec Command="git -C ../sassc describe --abbrev=4 --dirty --always --tags > sassc-version" />
-
+    <Exec Command="git -C .. describe --abbrev=4 --dirty --always --tags &gt; libsass-version" />
```

Comment:
Actually, it was automagically escaped by VS when I upgraded the project. Seems this is the legit way for executable commands: http://stackoverflow.com/a/2414096/863980 & https://msdn.microsoft.com/en-us/library/ms228186.aspx although `>` is not one of the special character which requires escaping; going by [their chart](https://msdn.microsoft.com/en-us/library/bb383819.aspx). :D